site/news/avatica-go-releases/index.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: news
+title: Avatica Go Releases
+permalink: /news/avatica-go-releases/
+author: all
+---
+{% comment %}
```

Comment:
Missing `<!--` and `-->` before and after the license comment? Saw this on the other files in this PR.